docs/src/dedicated-gen-3/overview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Dedicated Generation 3"
+weight: 1
+sidebarTitle: "Overview"
+layout: single
+description: |
+    Dedicated Generation 3 provides a scalable solution as an additional option on top of your existing Grid applications.
+    It provides redundant configuration with a minimum of three Virtual Machine instances. 
+---
+
+{{< description >}}
+
+Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.
+Dedicated Generation 3 works nearly identically to Grid environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 3?
```

Comment:
Also let’s always note that there are multiples reasons for which someone might choose dedicated over grid: 1. They need higher availability 2. They need bigger clusters than the maximum size offered by the grid. 3. They don’t want production data on a shared infrastructure. 4. They want to deploy somewhere where we don’t have a region.